Defs/ThingDefs_Misc/Weapons_Spotting.xml

```diff
@@ -33,7 +33,7 @@
       <SightsEfficiency>1.6</SightsEfficiency>
       <RangedWeapon_Cooldown>0.5</RangedWeapon_Cooldown>
       <Bulk>1.5</Bulk>
-      <Mass>3.00</Mass>
```

Comment:
The M22 binoculars used by the US military weighs 2.7 lb (~1.22 kg). The AN/PRC-152, a pretty modern handheld military radio, weighs 2.4 lb (1.09 kg). So, you can reduce the weight a bit if you really feel so compelled, but it's still not this light.

Reply:
Increased the weight to 2.25

Reply:
That's good, thanks!